--- origsrc/uemacs/Makefile	2012-07-17 07:09:17.000000000 -0500
+++ src/uemacs/Makefile	2012-07-17 07:42:46.490801100 -0500
@@ -56,6 +56,7 @@ LIBS=-lcurses			# SYSV
 LFLAGS=-hbx
 BINDIR=/usr/bin
 LIBDIR=/usr/lib
+STRIP=strip
 
 $(PROGRAM): $(OBJ)
 	$(E) "  LINK    " $@
@@ -72,7 +73,7 @@ clean:
 	$(Q) rm -f $(PROGRAM) core lintout makeout tags makefile.bak *.o
 
 install: $(PROGRAM)
-	strip $(PROGRAM)
+	$(STRIP) $(PROGRAM)
 	cp em ${BINDIR}
 	cp emacs.hlp ${LIBDIR}
 	cp emacs.rc ${LIBDIR}/.emacsrc
